performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,193 @@
+        randomBoundary = Integer.parseInt(PROPERTIES.getProperty("tableRandomBoundary","10"));
+        time = Long.parseLong(PROPERTIES.getProperty("tableTime", "100"));
+        keyNum = Integer.parseInt(PROPERTIES.getProperty("keyNum", "10"));
+        valueSize = Integer.parseInt(PROPERTIES.getProperty("valueSize", "1024"));
+        random = new Random(seed);
+        keyValueManager = new org.corfudb.performancetest.KeyValueManager(keyNum, valueSize);
+    }
+
+    /**
+     * Build a new CorfuTable using the given name.
+     * @param name the stream name.
+     * @return a new CorfuTable.
+     */
+    private CorfuTable<String, String> buildTable(String name) {
+        return runtime.getObjectsView().build()
+                .setTypeToken(new TypeToken<CorfuTable<String, String>>() {})
+                .setStreamName(name)
+                .open();
+    }
+
+    /**
+     * Generated a key value pair and put them in the CorfuTable.
+     * @param corfuTable the CorfuTable.
+     * @param numRequests how many times the function calls put().
+     */
+    private void putTable(CorfuTable corfuTable, int numRequests) {
+        for (int i = 0; i < numRequests; i++) {
+            String key = keyValueManager.generateKey();
+            String value = keyValueManager.generateValue();
+            corfuTable.put(key, value);
+        }
+    }
+
+    /**
+     * Getting an existing key from KeyValueManager and get the corresponding
+     * value from the CorfuTable.
+     * @param corfuTable the CorfuTable.
+     * @param numRequests how many times the function calls get().
+     */
+    private void getTable(CorfuTable corfuTable, int numRequests) {
+        for (int i = 0; i < numRequests; i++) {
+            String key = keyValueManager.getKey();
+            corfuTable.get(key);
+        }
+    }
+
+    /**
+     * Generate a new Thread which keeps calling putTable() and getTable() for a period.
+     * @param table The CorfuTable.
+     * @param start start time of a performance test.
+     * @return new Thread.
+     */
+    private Thread createThread(CorfuTable table, long start) {
+        return new Thread(() -> {
+            long duration = 0;
+            while ( duration <=  time) {
+                duration = (System.currentTimeMillis() - start) / MILLI_TO_SECOND;
+                try {
+                    putTable(table, random.nextInt(randomBoundary));
+                    getTable(table, random.nextInt(randomBoundary));
+                } catch (Exception e) {
+                    log.error("Operation failed with", e);
+                }
+            }
+        });
+    }
+
+    /**
+     * Performance test for using 1 CorfuTable and 1 Thread doing putTable and getTable operations.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @Test
+    public void CorfuTable1Thread1TableTest () throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)